azure-iot-hub-devicesdk/samples/send_telemetry_device.py

```diff
@@ -30,7 +32,13 @@ def connection_state_callback(status):
 # send 5 messages with a 1 second pause between each message
 for i in range(0, 5):
     print("sending message #" + str(i))
-    device_client.send_event("test_payload message " + str(i))
+    msg = Message("test weather payload " + str(i))
+    msg.message_id = uuid.uuid4()
+    msg.user_id = "AZIOT1234"
+    msg.custom_properties["temp"] = "71.89"
+    msg.custom_properties["humidity"] = "98%"
+    msg.custom_properties["windy"] = "yes"
```

Comment:
Is this a good example of what custom properties might be used for?  This seems more like something we traditionally put into the body of the message.